samples/xmlserializergenerator-instructions.md

```diff
@@ -0,0 +1,55 @@
+# Using Microsoft XmlSerializer Generator on .Net Core
+
+When a running application creates an instance of XmlSerializer, it generates a temporary assembly containing the implementation of the serializer. This happens once for each type for which an XmlSerializer is needed. To improve performance, we can use Microsoft XmlSerializer Generator to pre-generate code for an XmlSerializer for each specified type in a project and compile it into an assembly. This assembly will then be used by the application when creating an XmlSerializer instead of generating a temporary one.
+
+## Prerequisition
+
+You need to install the followings on your machine:
+* [.NET Core SDK daily builds](https://github.com/dotnet/cli#installers-and-binaries) 
+* [.NET Core runtime daily builds](https://github.com/dotnet/core-setup#daily-builds). 
+
+You can validate your .NET Core SDK version by typing `dotnet --info`.
+
+## Instructions
+
+Here are the step by step instructions how to create a dotnet project and install the XmlSerializer Generator for that project.
+
+1. Create a library project that contains your custom data types. E.g. create a library named MyData using the CLI: `dotnet new library --name MyData`
```

Comment:
Wouldn't you already have this library and "customer data types"? Otherwise, it will be quite non-obvious what to do. Also, I suspect that these data types are really just classes, right? Unless I misunderstand, "MyData" would be an odd name for a library.

Reply:
The example I am giving here contains a console app and a library project. The library project contains all the custom data I need.